js/compiler/compiler.go

```diff
@@ -210,9 +211,17 @@ func (c *Compiler) compileImpl(
 	state := compilationState{srcMap: srcMap, compiler: c, wrapped: wrap}
 	if wrap {
 		conditionalNewLine := ""
-		if strings.Contains(code, "//# sourceMappingURL=") {
+		if index := strings.LastIndex(code, "//# sourceMappingURL="); index != -1 {
 			// the lines in the sourcemap (if available) will be fixed by increaseMappingsByOne
 			conditionalNewLine = "\n"
+			newCode, err := state.updateInlineSourceMap(code, index)
+			if err != nil {
+				c.logger.Warn("while compiling %q we couldn't update it's inline sourcemap which might lead "+
```

Comment:
It is not clear to me as a user, if I can fix it in some way. Do we have a suggestion? Or the answer is `there isn't much you can do`.

Reply:
I would argue this warning is mostly so that when they report an issue - they report it with this warning